articles/101_build_tool.md

```diff
@@ -166,19 +170,37 @@ It was developed after `catkin_make` to address the problems involved with build
 The tool only supports CMake-based packages and builds each package in topological order using the command sequence common for CMake packages: `cmake`, `make`, `make install`.
 While each package can parallelize the build of its targets the packages are processed sequentially even if they are not (recursive) dependencies of each other.
 
+`catkin_make_isolated` supports building the following packages:
+
+- ROS 1 `catkin` package with a `package.xml` file.
+- Plain CMake packages with a `package.xml` file.
+
 ### catkin_tools
 
 [catkin_tools](https://catkin-tools.readthedocs.io/) is provided by a standalone Python package used to build ROS 1 packages.
 It was developed after `catkin_make` / `catkin_make_isolated` to build packages in parallel as well as provide significant usability improvements.
 The tool supports building CMake packages and builds them in isolation as well as supports parallelizing the process across packages.
 
+`catkin_make_isolated` supports building the following packages:
+
+- ROS 1 `catkin` package with a `package.xml` file.
+- Plain CMake packages with a `package.xml` file.
```

Comment:
The headline is certainly a copy-n-paste error. Good catch :+1:

The supported build types are still correct, right?